pages/common/kubectl-autoscale.md

```diff
@@ -0,0 +1,16 @@
+# kubectl autoscale
+
+> Creates an autoscaler to intelligently scale pod count based on kubernetes cluster demands.
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#autoscale>.
+
+- Usage:
+
+`kubectl autoscale (-f FILENAME | TYPE NAME | TYPE/NAME) [--min=MINPODS] --max=MAXPODS [--cpu-percent=CPU]`
+
```

Comment:
```suggestion
```
We do not use this format. Create direct commands that instruct with doing specific things.